extensions/wikia/Development/SpecialDevBoxPanel/Special_DevBoxPanel.php

```diff
@@ -242,6 +242,7 @@ function wfDevBoxResourceLoaderGetConfigVars( &$vars ) {
 	global $wgDevelEnvironment;
 
 	$vars['wgDevelEnvironment'] = $wgDevelEnvironment;
+	$vars['wgWikiaDatacenter'] = $wgWikiaDatacenter;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "$wgWikiaDatacenter" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1481)